test/integration/basic_test.lua

```diff
@@ -42,6 +42,20 @@ g.cluster = cartridge_helpers.Cluster:new({
     },
 })
 
+local function set_sections(sections)
```

Comment:
Why not move it to utils? As i see it's used in cleanup too

Reply:
fixed